accepted/2021/statics-in-interfaces/README.md

```diff
@@ -884,22 +948,53 @@ namespace System
         static abstract TSelf Round<TInteger>(TSelf x, TInteger digits, MidpointRounding mode)
             where TInteger : IBinaryInteger<TInteger>;
 
-        // IEEE defines n to be an integral type, but not the size
+        static abstract TSelf Truncate(TSelf x);
+    }
 
-        static abstract TSelf ScaleB<TInteger>(TSelf x, TInteger n)
-            where TInteger : IBinaryInteger<TInteger>;
+    public interface IFloatingPointIeee754<TSelf>
+        : IExponentialFunctions<TSelf>,
+          IFloatingPoint<TSelf>,
+          IHyperbolicFunctions<TSelf>,
+          ILogarithmicFunctions<TSelf>,
+          IPowerFunctions<TSelf>,
+          IRootFunctions<TSelf>,
+          ITrigonometricFunctions<TSelf>
+        where TSelf : IFloatingPointIeee754<TSelf>
```

Comment:
Based on the feedback, `IFloatingPoint` should be split into itself + `IFloatingPointIeee754`. This ensures that `decimal` can appropriately implement it.

Likewise, a few of the "function groups" should have their own interfaces to allow users to more readily support these on their own types, even if not IEEE 754 compliant.

This includes:
* `Exponential`
* `Hyperbolic Trigonometric`
* `Logarithmic`
* `Powers`
* `Roots`
* `Trigonometric`